package.json

```diff
@@ -56,6 +56,11 @@
       "type": "object",
       "title": "sway-lsp",
       "properties": {
+        "sway-lsp.path": {
+          "description": "Select the path to the Sway language server.",
```

Comment:
I think we should be a bit more descriptive here so users know that this is an optional field, and that it's only to be used if you want the plugin to load from a custom location, otherwise the binary if $PATH will be used.

Reply:
Agreed, maybe something like: "Optionally override the path to the Sway language server executable. If empty, the extension will use the `forc-lsp` executable to which your $PATH resolves."